src/GitVersion.Core/Model/VersionVariables.cs

```diff
@@ -170,7 +172,15 @@ public static VersionVariables FromJson(string json)
             return FromDictionary(variablePairs);
         }
 
-        public static VersionVariables FromFile(string filePath, IFileSystem fileSystem)
+        public static VersionVariables FromFile(string filePath, IFileSystem fileSystem, ILog log)
+        {
+            var retryOperation = new OperationWithExponentialBackoff<IOException, VersionVariables>(new ThreadSleep(), null, () => FromFileInternal(filePath, fileSystem), maxRetries: 6);
```

Comment:
Question, why maxRetries: 6? why not use the default 5?

Reply:
I took this from the example from the GitVersion Cache that you shared earlier. I thought there was a reason for the non-default value. 5 should be Ok here. Just let me know and I'll adjust.

Reply:
Use the default

Reply:
Done